graphstorm-processing/tests/resources/small_heterogeneous_graph/gsprocessing-config.json

```diff
@@ -104,7 +104,7 @@
                 "data": {
                     "format": "csv",
                     "files": [
-                        "edges/movie-included_in-genre.csv"
+                        "./edges/movie-included_in-genre.csv"
```

Comment:
Shall we leave one as `./` and keep another as the same?

Reply:
We have other paths in the doc that don't include a `./` like 

```
 "data": {
                    "format": "csv",
                    "files": [
                        "nodes/user.csv"
                    ],
                    "separator": ","
                },
```

in lines 28-34